tests/model/test_core.py

```diff
@@ -1786,15 +1782,24 @@ def test_copy_model(self, copy_method) -> None:
                 samples=1, random_seed=42
             )
 
-        simple_model_prior_predictive_mean = simple_model_prior_predictive["prior"]["y"].mean(
-            ("chain", "draw")
-        )
-        copy_simple_model_prior_predictive_mean = copy_simple_model_prior_predictive["prior"][
+        simple_model_prior_predictive_val = simple_model_prior_predictive["prior"]["y"].values
+        copy_simple_model_prior_predictive_val = copy_simple_model_prior_predictive["prior"][
```

Comment:
Just compare directly, no need to assign to separate variables, that are almost as verbose as the way they are accessed